views/peeps/new.erb

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel=”stylesheet” type=”text/css” href=”/style.css” />
+<style>
+.text {
+  text-align: center;
+}
+</style>
+</head>
+
+<body>
+<div class="text">
+<h1> Give us a peep into your day/thoughts/whatever </h1>
+
+
+<form action='/peeps' method='post'>
+  <input type='text' size="100" name='text' 
+  minlength='1' maxlength='240'placeholder='MAX LENGTH = 240 Characters'>
```

Comment:
excellent putting a max length in here, I had a max length in my database but not here which could definitely lead to complications. great idea!